tests/optional_kernel_features/kernel_features_common.h

```diff
@@ -38,7 +49,7 @@ namespace kernel_features_common {
  */
 template <typename T, sycl::aspect aspect>
 [[sycl::device_has(aspect)]] SYCL_EXTERNAL void
-use_feature_function_external_decorated();
+use_feature_function_external_decorated(const sycl::accessor<bool, 1> &acc);
```

Comment:
I thought that the coding style for SYCL and the CTS is to have `&` and `*` part of the type everywhere.
```suggestion
use_feature_function_external_decorated(const sycl::accessor<bool, 1>& acc);
```

Reply:
Agree with you but clang-format.